src/backend/distributed/operations/shard_split.c

```diff
@@ -377,9 +405,12 @@ SplitShard(SplitMode splitMode,
 	else if (PlacementMovedUsingLogicalReplicationInTX)
 	{
 		ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-						errmsg("multiple shard movements/splits via logical "
+						errmsg("splitting/moving multiple shard via logical "
```

Comment:
multiple shard -> multiple shards

does the error make sense for create_distributed_table_concurrently? (not sure how hard we should try)